### PR TITLE
feat(docs): refactor guide on graphql

### PR DESCRIPTION
**Pull request by yoonhyejin**
## Checklist

- [ ] The PR conforms to DataHub's [Contributing Guideline](https://github.com/datahub-project/datahub/blob/master/docs/CONTRIBUTING.md) (particularly [Commit Message Format](https://github.com/datahub-project/datahub/blob/master/docs/CONTRIBUTING.md#commit-message-format))
- [ ] Links to related issues (if applicable)
- [ ] Tests for the changes have been added/updated (if applicable)
- [ ] Docs related to the changes have been added/updated (if applicable). If a new feature has been added a Usage Guide has been added for the same.
- [ ] For any breaking change/potential downtime/deprecation/big changes an entry has been made in [Updating DataHub](https://github.com/datahub-project/datahub/blob/master/docs/how/updating-datahub.md)

resolves DR-359

## Demo
https://docs-website-7dtkfrh4q-acryldata.vercel.app/docs/api/graphql/how-to-set-up-graphql

### CODE REVIEW

**Comment by hsheth2** [comment]
@yoonhyejin for the SDK section, I'm adding this PR to make it easier via python https://github.com/datahub-project/datahub/pull/7753

File: .pre-commit-config.yaml
Comment (gabe-lyons) [inline .pre-commit-config.yaml:24]: what is the significance of this removal?

File: docs-website/sidebars.js
Comment (gabe-lyons) [inline docs-website/sidebars.js:407]: are generating access tokens gone or just getting moved?

File: docs/api/graphql/getting-started.md
Comment (gabe-lyons) [inline docs/api/graphql/getting-started.md:61]: nice !

File: .pre-commit-config.yaml
Comment (yoonhyejin) [reply .pre-commit-config.yaml:24]: just removed unnecessary pre-commit checks, but tbh this is unrelated to this PR.

File: docs/api/tutorials/adding-tags.md
Comment (hsheth2) [inline docs/api/tutorials/adding-tags.md:39]: can we use `graphql` here so that the highlighting looks correct?

**Review by gabe-lyons — APPROVED** [review]
stamping to unblock, would be good to address Harshal's syntax highlighting before merging

**Pull request merged**
